task/01-strings-tasks.js

```diff
@@ -114,7 +120,11 @@ function removeLeadingAndTrailingWhitespaces(value) {
  *   'cat', 3 => 'catcatcat'
  */
 function repeatString(value, count) {
-    throw new Error('Not implemented');
+    let result = ''
+    for (let i = 0; i < count; i++) {
+        result += value
+    }
+    return result
```

Comment:
This solution will work, but we should preferably use `String` methods, such as `repeat` in this case